services/notifications/setup.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import re
+import sys
+from pathlib import Path
+
+from setuptools import find_packages, setup
+
+
+def read_reqs(reqs_path: Path) -> set[str]:
+    return {
+        r
+        for r in re.findall(
+            r"(^[^#\n-][\w\[,\]]+[-~>=<.\w]*)",
+            reqs_path.read_text(),
+            re.MULTILINE,
+        )
+        if isinstance(r, str)
+    }
+
+
+CURRENT_DIR = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
+
+NAME = "simcore-service-notifications"
+VERSION = (CURRENT_DIR / "VERSION").read_text().strip()
+AUTHORS = ("Andrei Neagu (GitHK)",)
+DESCRIPTION = "Service for executing commands on docker nodes"
+
+PROD_REQUIREMENTS = tuple(
+    read_reqs(CURRENT_DIR / "requirements" / "_base.txt")
+    | {
+        "simcore-models-library",
```

Comment:
I think ideally the `PROD_REQUIREMENTS`  should be fully listed in `_base.txt`, I think those three additional requirements are not required and can be removed

Reply:
these are required, otehrwise the sicmore-models-library package will not be detected at runtime when you install the library with `pip install .`

Reply:
Your argument looks valid. I think yhe corect file to use instead of all this is just `prod.txt` since it should contain all the required packages. @pcrespov @sanderegg any input on this?

Reply:
At the time we did this, setuptools did not allow installing local libraries (i.e. python libraries in `packages/`) from a requirements file using **relative** paths

new uv and package manager might have improve this. For now we have to be a bit redundant until we get thisone done https://github.com/ITISFoundation/osparc-simcore/issues/4433